lib/generators/rest_api_generator/helpers.rb

```diff
@@ -17,5 +17,14 @@ def editable_attributes
         Rails::Generators::GeneratedAttribute.new(column.name.to_s, column.type.to_s)
       end
     end
+
+    def versioning?
```

Comment:
I dont know if versiong is the best name for this feature, we would like to be like a custom path for the model and controller that can be used for versioning, maybe stay with name scope is better

Reply:
done